cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/WithTest.scala

```diff
@@ -41,32 +41,64 @@ class WithTest extends DocumentingTest {
         #  (b)-[:BLOCKS]->(d)""".stripMargin('#'))
     synopsis("The `WITH` clause allows query parts to be chained together, piping the results from one to be used as starting points or criteria in the next.")
     note {
-      p("It is important to note that `WITH` affects variables in scope. Any variables not included in the `WITH` clause are not carried over to the rest of the query.")
+      p(
+        """It is important to note that `WITH` affects variables in scope. Any variables not included in the `WITH` clause are not carried over to the rest of the query.
+          |The wildcard `*` can be used to include all variables that are currently in scope.
+          |""".stripMargin)
     }
     p(
       """* <<with-introduction, Introduction>>
+        |* <<with-introduce-variables, Introducing variables for expressions>>
+        |* <<with-wildcard, Using the wildcard to carry over variables>>
         |* <<with-filter-on-aggregate-function-results, Filter on aggregate function results>>
         |* <<with-sort-results-before-using-collect-on-them, Sort results before using collect on them>>
         |* <<with-limit-branching-of-path-search, Limit branching of a path search>>
       """.stripMargin)
     section("Introduction", "with-introduction") {
       p(
         """Using `WITH`, you can manipulate the output before it is passed on to the following query parts.
-          |The manipulations can be of the shape and/or number of entries in the result set.""".stripMargin)
+          |Manipulations can be done to the shape and/or number of entries in the result set.""".stripMargin)
```

Comment:
I find the word "shape" a bit vague here. And is the "result set" what we call "row" elsewhere?

Also a small (opinionated) suggestion:
```suggestion
          |Manipulations can be done to the shape and/or to the number of entries in the result set.""".stripMargin)
```